### ISSUE TITLE
Enhancement request:  Allow verification of backups after they have been created.

### ISSUE THREAD

**Issue by jharris1993** [procount/pinn#495]
Issue:

There is currently no way to verify if a backup is valid, (at least checksum wise), after it has been created aside from restoring it somewhere.

It would be nice to be able to select an existing backup and have PINN at least verify the checksum of the tarballs contained therein.

Thanks!

**Comment by KilliK69** [comment]
I agree, this feature would be nice. For now, after I finish a backup, I have to boot to Raspbian and use peazip to test the integrity of the .tar.gz files.
